lib/hackerone/client/report.rb

```diff
@@ -75,6 +75,32 @@ def activities
         end
       end
 
+      def program
+        @program || Program.find(relationships[:program][:data][:attributes][:handle])
+      end
+
+      def assign_to_user(name)
+        fail(
```

Comment:
That's a fair point. In general, any error messages should include information about the specific failure [like you did here](https://github.com/oreoshake/hackerone-client/pull/16/files#diff-5db89a988dfc98aa51b9525b199db48fR138).